src/app/dataset/depublication/depublication.component.ts

```diff
@@ -228,6 +245,15 @@ export class DepublicationComponent implements OnDestroy {
     }
   }
 
+  onDepublishDataset(): void {
+    this.depublications.depublishDataset(this._datasetId).subscribe((success: boolean) => {
+      if (success) {
+        this.depublicationComplete = true;
```

Comment:
Maybe set it to false before triggering the depublication? Just in case you're doing it more than once... (which really shouldn't happen, but you never know)

Reply:
I'm using this.depublicationComplete to show the message in the grid as to how many records were depublished.  It's true that this will need cleared, but there's no obvious time to do it.  (perhaps when they leave the tab and return to it?)

Reply:
Well, if the only result is that the message just stays until it is replaced by the next one, I guess it is fine. In reality this is really not very likely.